index.js

```diff
@@ -13,14 +13,15 @@ const defaults = {
   reporters: null,
   reporterDefaults: {
     filter: [],
+    throttle: {},
```

Comment:
lets do two separate options to make it more explicit.

`throttle: false, throttleBasedOnTags: false`